a/apache-calcite-avatica/build_info.json

```diff
@@ -0,0 +1,10 @@
+{
+    "package_name":"calcite-avatica",
```

Comment:
@shubham-garud please change it to `apache-calcite-avatica`

Reply:
Sure.